src/utils/displayDocumentList.js

```diff
@@ -0,0 +1,45 @@
+import { getItem } from "./storage.js";
+
+export const displayDocumentList = (documents) => {
+  return `<ul>
+          ${documents
+            .map((document) => {
```

Comment:
p3: 이런식으로 미리 빼두면 더 편할 것 같아요 : )
```suggestion
            .map(({id,title,documents}) => {
```